crates/xwt-core/src/session/datagram.rs

```diff
@@ -4,6 +4,16 @@ use core::future::Future;
 
 use crate::utils::{maybe, Error};
 
+pub trait MaxDatagramSize: maybe::Send {
```

Comment:
```suggestion
pub trait MaxSize: maybe::Send {
```

Needless to mention `Datagram` in the `datagram` mod...

Reply:
That said - maybe we should specify this is the max size of the outgoing datagram at the name?

Reply:
I think that would make the fn name too verbose, and none of the implementations seem to follow that either:
- https://developer.mozilla.org/en-US/docs/Web/API/WebTransportDatagramDuplexStream/maxDatagramSize - uses `maxDatagramSize`
- https://docs.rs/wtransport/latest/wtransport/connection/struct.Connection.html#method.max_datagram_size - uses `max_datagram_size`

I think renaming the trait to `datagram::MaxSize` is fine, but I want to keep the fn name as `max_datagram_size`

Reply:
Makes sense, let's move forward with this `fn` name then, but with the rename of the trait